docs/architecture/adr-004-tips-contract.md

```diff
@@ -170,6 +165,7 @@ pub struct TipsResponse {
 pub struct Tip {
   pub sender: Addr,
```

Comment:
I think it would be useful to have `block_height` field. The user would like to knwo when the tip happens.

Reply:
I do agree, @bragaz what do you think?

Reply:
Yes agree too. @manu0466